config/default/field.field.node.hs_basic_page.field_hs_page_components.yml

```diff
@@ -42,6 +44,9 @@ settings:
       hs_carousel:
         weight: 15
         enabled: false
+      hs_gradient_hero:
+        weight: 21
+        enabled: false
```

Comment:
This configuration is present for hs_gradient_hero.  We probably don't want this to be showing up in here yet.